tests/logging/check_logs_table.sh

```diff
@@ -140,32 +144,60 @@ check_system_history_permissions() {
         '.state == "Failed"'
 
-    # Command 3: User 'a:123' attempts to drop 'system_history.access_history' table (expected to succeed)
+    # Command 3: User 'a:123' drop 'system_history.access_history' table (expected to succeed)
     execute_and_verify \
-        "User 'a:123' attempts to drop 'system_history.access_history' table" \
+        "User 'a:123' drop 'system_history.access_history' table" \
         "a:123" \
         '{"sql": "drop table system_history.access_history"}' \
         '.state == "Succeeded"'
 
-    # Command 4: User 'root:' attempts to grant ownership on 'system_history.*' (expected to fail)
+    # Command 4: User 'root:' grant ownership on 'system_history.*' (expected to fail)
     execute_and_verify \
-        "User 'root:' attempts to grant ownership on 'system_history.*'" \
+        "User 'root:' grant ownership on 'system_history.*'" \
         "root:" \
         '{"sql": "grant ownership on system_history.* to role ra"}' \
         '.state == "Failed"'
 
-    # Command 5: User 'root:' attempts to grant ownership on 'system_history.query_history' (expected to fail)
+    # Command 5: User 'root:' grant ownership on 'system_history.query_history' (expected to fail)
     execute_and_verify \
-        "User 'root:' attempts to grant ownership on 'system_history.query_history'" \
+        "User 'root:' grant ownership on 'system_history.query_history'" \
         "root:" \
         '{"sql": "grant ownership on system_history.query_history to role ra"}' \
         '.state == "Failed"'
-    # Command 6: User 'a:123' attempts to select 'system_history.query_history' table (expected to succeed)
+    # Command 6: User 'a:123' select 'system_history.query_history' table (expected to succeed)
         execute_and_verify \
-            "User 'a:123' attempts to query 'system_history.query_history' table" \
+            "User 'a:123' query 'system_history.query_history' table" \
             "a:123" \
             '{"sql": "select count() from system_history.query_history"}' \
             '.state == "Succeeded"'
+
+    # Command 7: User 'a:123' drop stage 'log_1f93b76af0bd4b1d8e018667865fbc65' table (expected to failed)
+            execute_and_verify \
+                "User 'a:123' drop stage log_1f93b76af0bd4b1d8e018667865fbc65" \
+                "a:123" \
+                '{"sql": "drop stage log_1f93b76af0bd4b1d8e018667865fbc65"}' \
+                '.state == "Failed"'
+
+    # Command 8: User 'a:123' copy into @log_1f93b76af0bd4b1d8e018667865fbc65 from (select * from system.one) (expected to failed)
+                execute_and_verify \
+                    "User 'a:123' copy into @log_1f93b76af0bd4b1d8e018667865fbc65 from (select * from system.one);" \
+                    "a:123" \
+                    '{"sql": "copy into @log_1f93b76af0bd4b1d8e018667865fbc65 from (select * from system.one);"}' \
+                    '.state == "Failed"'
```

Comment:
better replace `.state == "Failed"`  with some key words of error message, or error code if the code is specific enough